runtime/src/main/java/org/corfudb/runtime/view/stream/BackpointerStreamView.java

```diff
@@ -178,5 +185,10 @@ protected boolean discoverAddressSpace(final UUID streamId,
 
         return !queue.isEmpty();
     }
+
+    @Override
+    protected long getMaxGlobalFromMaxEntries(int maxEntries) {
+        return Address.MAX;
+    }
```

Comment:
Instead of returning an incorrect value here and relying on remainingAtMost to throw an exception I think its cleaner to just move UnsupportedOperationException to a default implementation in the IStreamView interface.

Reply:
Done.